packages/react-storage/src/components/StorageBrowser/views/LocationActionView/__tests__/CopyFilesControls.spec.tsx

```diff
@@ -15,13 +17,29 @@ const TEST_ACTIONS = {
 
 jest.spyOn(TempActions, 'useTempActions').mockReturnValue(TEST_ACTIONS);
 const useCopyViewSpy = jest.spyOn(UseCopyViewModule, 'useCopyView');
+const useDataSpy = jest.spyOn(AmplifyReactCore, 'useDataState');
 
 describe('CopyFilesControls', () => {
   const onExitMock = jest.fn();
   const onActionCancelMock = jest.fn();
   const onActionStartMock = jest.fn();
   const onSetDestinationList = jest.fn();
 
+  beforeAll(() => {
+    useDataSpy.mockReturnValue([
+      {
+        data: {
+          items: [{ id: '1', key: 'Location A', type: 'FOLDER' }],
+          nextToken: undefined,
+        },
+        message: '',
+        hasError: false,
+        isLoading: false,
+      },
+      jest.fn(),
+    ]);
+  });
```

Comment:
gets rid of `act` warnings

Reply:
If it works for you, can incorporate the updates in this file in to https://github.com/aws-amplify/amplify-ui/pull/5998 to reduce merge conflicts